plugin/camelsnek.vim

```diff
@@ -15,8 +15,20 @@ if !exists('g:camelsnek_alternative_camel_commands')
   let g:camelsnek_alternative_camel_commands = 0
 end
 
-if !exists('g:camelsnek_i_am_an_old_fart_with_no_sense_of_humour_or_internet_culture')
-  let g:camelsnek_i_am_an_old_fart_with_no_sense_of_humour_or_internet_culture = 0
+if !exists('g:camelsnek_no_fun_allowed')
+  let g:camelsnek_no_fun_allowed = 0
+end
+
+" Handles iskeyword update to make kebab editing easy.
+if !exists('g:camelsnek_iskeyword_override')
+  let g:camelsnek_iskeyword_override = 1
```

Comment:
```suggestion
  let g:camelsnek_iskeyword_override = 0
```

Reply:
I like this suggestion, forgot I had it on by default. It's simpler than alternatives to selecting by word above.